src/btree/multi.ts

```diff
@@ -5,20 +5,22 @@ import { PageFile } from "./pagefile";
 const PAGE_SIZE_BYTES = 4096;
 export const maxUint64 = 2n ** 64n - 1n;
 
+const globalMetaPageCache = new Map<
```

Comment:
resolved all comments above -- added global meta page cache.

It caches properly now